manage/installing/installation.rst

```diff
@@ -125,11 +125,11 @@ Download the latest Plone unified installer
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 Download from `the plone.org download page <https://plone.org/download>`_ to your server using wget command. Curl also works.
```

Comment:
We still have to decide how to deal with the "downloads" page on the new plone.org. Since we deprecated PSC we need to design and implement a brand new one. I agree with @jensens however, it would be great if it's the least human dependant possible. Let's say that we have some way to detect which are the last versions and update links and content accordingly.

How we could achieve that? Any ideas?